spec/models/video_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+ describe Video do
+  it { should belong_to(:category)}
+  it { should validate_presence_of(:title)}
+  it { should validate_presence_of(:description)}
+
+  it "cannot find a single video" do
```

Comment:
Or at least for code review - it is just a spam and if you have unfinished work it belongs to a different branch.